### PR TITLE
Feature/add timeout for graphite plugin

### PR DESCRIPTION
**Pull request by pnuckowski**

### CODE REVIEW

File: alamo_worker/cli.py [hunk 1]
```diff
@@ -111,7 +111,12 @@ def execute(self):
         while self.scheduler_queue.zmq_socket.poll():
             check = self.scheduler_queue.zmq_socket.recv_json()
             result = self.manager.dispatch(check)
-            self.send(result)
+            if result:
+                self.send(result)
+            else:
+                logger.error(
+                    'Check %s wasn"t dispatched properly!', check['id']
```
Comment (deejay1) [inline alamo_worker/cli.py]: '' in strings are evil maybe switch to normal " " ?

File: alamo_worker/plugins/__init__.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
-import pytz
 from datetime import datetime
 from configparser import NoSectionError
 
+import pytz
+
```
Comment (deejay1) [inline alamo_worker/plugins/__init__.py]: IMHO you can get rid of that empty line here

File: alamo_worker/cli.py (hunk 1 above)
Comment (h00dy) [inline alamo_worker/cli.py]: +1

**Comment by deejay1** [comment]
:octocat:

**Comment by h00dy** [comment]
:gun:

**Pull request merged**